### PR TITLE
Dockerfile & SSL "Emulating"

### PR DESCRIPTION
**Pull request by cactushydrocodone**
Hello,
I want to use a "https://" url for my pic uploads from telegram.
At the moment, when i send a picture from telegram to irc, the url looks like this:
http://mycoolsite.com/kitties.png
my site is HTTPS only so i want to use HTTPS.
My idea was to just "cosmetically" change the url to 
https://mycoolsite.com/kitties.png

Also i created a small dockerfile.

Feel free to edit/fix/make it more qt

### CODE REVIEW

File: src/web_backend.py [hunk 1]
```diff
@@ -50,7 +50,15 @@ def __init__(self, config):
 			self.subdirs = config["use_subdirs"]
 			# Used by download_and_serve():
 			self.webpath = tempfile.mkdtemp()
-			self.baseurl = "http://%s:%d" % (extern, port)
+			#SSL
+			self.ssl == config["ssl"]
```
Comment (sfan5) [inline src/web_backend.py:54]: ok this is epic

Comment (cactushydrocodone) [reply src/web_backend.py:54]: HEY IM NOT A PROFESSIONAL
atleast recommend me somthing

Comment (sfan5) [reply src/web_backend.py:54]: `==` is a comparison not an assigment
your `if` statements are also missing colons

**Comment by sfan5** [comment]
implemented in 69a8175

**Pull request closed without merging**